pkg/v1/cli/signals/signals.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2022 VMware, Inc. All Rights Reserved.
```

Comment:
This package looks like generally useful outside of the CLI. Should this be in `pkg/v1` instead of `pkg/v1/cli`?